src/site/_includes/path.njk

```diff
@@ -2,7 +2,9 @@
 layout: layout
 ---
 
-{% set topics = path.topics | removeDrafts(lang) %}
+{% set path = paths[pathName] %}
```

Comment:
Since all of the path data lives in _data/paths I think we could use [Eleventy's pagination](https://www.11ty.dev/docs/pagination/) to generate the path pages (/accessible/, /secure/, etc). Basically the same thing we do to generate the authors and the tags pages.

path.njk would set the pagination size to 1, and the data to _data/paths/all.
@MichaelSolati does that sound right to you?

Reply:
Possibly, I don't think pagination works well with data files, but we could probably "hack" it and bypass the use of a collection with an 11ty.data.js file.

Reply:
hm that seems weird. It just can't recognize the data in scope or something?

Reply:
seems to be supported? https://www.11ty.dev/docs/pagination/#paginate-a-global-or-local-data-file

Reply:
threw together a quick example
https://glitch.com/edit/#!/pale-safe-chance

Reply:
I don't really see what would be a benefit of using pagination instead of the data file directly?

Reply:
You would still use the data files, but you could delete files like: `content/en/accessible/index.jk` and instead have a single file that outputs all of our path files. It also means if we change the path template we only need to do it in one place, instead of having to update `content/en/accessible/index.jk`, `content/en/secure/index.jk`, etc.

Reply:
I'd like to keep the .njk files to be honest. They do not include any markup, only frontmatter anyways. They will be useful to keep translated content for each path in language folders (e.g. a translated title and description).